src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/MessageBody.cs

```diff
@@ -246,8 +246,8 @@ internal static unsafe void ParseConnection(StringValues connection, out bool is
                                 (*++ch | 0x20) == 'v' &&
                                 (*++ch | 0x20) == 'e')
                             {
-                                isKeepAlive = true;
                                 ch++;
+                                isKeepAlive = ch == end || *ch == ',' || *ch == ' ';
```

Comment:
The space or comma char delimits the token, that's all I care about here. The following spaces/commas will be skipped later.